biggraphite/drivers/elasticsearch.py

```diff
@@ -507,8 +507,8 @@ def delete_directory(self, name):
         query.delete()
 
     # TODO (t.chataigner) Add unittest.
-    def _search_metrics_from_components(self, glob, components):
-        search = self._create_search_query().source('name')
+    def _search_metrics_from_components(self, glob, components, start_time=None, end_time=None):
+        search = self._create_search_query(start_time, end_time).source('name')
```

Comment:
What do you think about giving a "Search" object to `_search_metrics_from_components` and letting the caller call `_create_search_query()` ? This way you would not have to give start_time/end_time to all these functions

Reply:
Done.